CommandLine.c

```diff
@@ -363,7 +363,7 @@ int CommandLine_run(int argc, char** argv) {
    CRT_init(settings, flags.allowUnicode);
 
    MainPanel* panel = MainPanel_new();
-   ProcessList_setPanel(pl, (Panel*) panel);
+   Table_setPanel(&pl->super, (Panel*) panel);
```

Comment:
If that was the case it would be a problem already, independent of Dynamic Screens, since this is how processes and (multiple) screen panels are setup today.  Changing this model would be a major deviation from the current code.  I'm inclined to leave this to a possible follow-up commit, if anyone wants to rework this: it's not a new issue that has been introduced here.